packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -370,7 +370,10 @@ describe('backupNg', () => {
         [scheduleTempId]: DEFAULT_SCHEDULE,
       },
       settings: {
-        ...defaultBackupNg.settings,
+        '': {
+          bypassVdiChainsCheck: true,
```

Comment:
This is not obvious: you forgot it the first time.

You need to add doc somewhere (maybe simple a comment) explaining why that's necessary!

Reply:
In https://github.com/vatesfr/xen-orchestra/tree/master/packages/xo-server-test#best-practices?

Reply:
As you wish.